ffbs-parker-nextnode/files/lib/gluon/ebtables/399-localnode

```diff
@@ -0,0 +1,12 @@
+local client_bridge = require 'gluon.client_bridge'
+local site = require 'gluon.site'
+local next_node = site.next_node({})
+local macaddr = client_bridge.next_node_macaddr()
+
+if next_node.ip4 then
+        rule('PREROUTING -p IPv4 -d ! ' .. macaddr .. ' --ip-dst ' .. site.next_node.ip4() .. ' -j dnat --to-dst ' .. macaddr .. ' --dnat-target ACCEPT', 'nat')
```

Comment:
Is the usage of `next_node.ip4` and `site.next_node.ip4()` intentional? Same for ip6.